dev_guide/expanding_persistent_volumes.adoc

```diff
@@ -0,0 +1,39 @@
+[[expanding_persistent_volumes]]
+= Expanding Persistent Volumes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[resizing-pv]]
+== Resizing Persistent Volumes
+
+You can expand persistent volume claims (PVC) online from {product-tile} for
+Containerized GlusterFS, Cinder, and Google Compute Engine (GCE) Persistent Disks (PD) by creating a storage class with
```

Comment:
Is the `Containarized` word to emphasis CNS? I do not think it is needed, we should just say `GlusterFS`